### PR TITLE
Support Uniswap v3 price feed

### PR DESCRIPTION
**Pull request by bun919tw**

### CODE REVIEW

**Comment by bun919tw** [comment]
Multiprice oracle: https://etherscan.io/address/0x1f53a354b726dcd1fc8053182a46de6b9995740f

File: contracts/oracles/UniswapV3PriceFeed.sol [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import "../interfaces/IPriceFeed.sol";
+
+contract UniswapV3PriceFeed is IPriceFeed {
+    address public registry;
+    address public multiPriceOracle;
+    address public token;
+
+    address public constant weth = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    uint public constant amountIn = 1e18;
+    uint32 public constant twapPeriod = 86400; // 1 day
```
Comment (jeremyHD) [inline contracts/oracles/UniswapV3PriceFeed.sol]: how about
```suggestion
    uint32 public constant twapPeriod = 1 days;
```
https://docs.soliditylang.org/en/v0.8.6/units-and-global-variables.html#time-units

File: contracts/oracles/UniswapV3PriceFeed.sol [hunk 2]
```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/interfaces/FeedRegistryInterface.sol";
+import "@chainlink/contracts/src/v0.8/Denominations.sol";
+import "../interfaces/IMultipriceOracle.sol";
+import "../interfaces/IPriceFeed.sol";
+
+contract UniswapV3PriceFeed is IPriceFeed {
```
Comment (jeremyHD) [inline contracts/oracles/UniswapV3PriceFeed.sol:10]: Should we name this as WETH pair related?

**Review by jeremyHD — APPROVED** [review]

File: contracts/oracles/UniswapV3PriceFeed.sol (hunk 2 above)
Comment (bun919tw) [reply contracts/oracles/UniswapV3PriceFeed.sol:10]: I would like to keep the name clean.

**Pull request merged**